### PR TITLE
[Go] config file

### PR DESCRIPTION
**Pull request by tehsphinx**
@ErikSchierboom: I removed the float dependency from strings again. Even though we format a float value in there it just doesn't fit in the general context of exercises. The strings exercise would be available at `basics` > `numbers` > `floating-points` > `strings`. With that the whole point of an early strings exercise is gone, again. I'd rather have a small paragraph on floats in the intro, instead.

### CODE REVIEW

File: languages/go/config.json [hunk 1]
```diff
@@ -61,25 +61,25 @@
...
         "concepts": ["constants"],
         "prerequisites": ["strings", "numbers"]
       },
       {
         "slug": "context",
-        "uuid": "96aaabad-b8b7-431c-ae59-abb813efd272",
+        "uuid": null,
         "concepts": ["context"],
         "prerequisites": ["time-and-duration", "channels", "select"]
       },
```
Comment (oanaOM) [inline languages/go/config.json:73]: @tehsphinx I think we should also change the prerequisites here from `time-and-duration ` to `duration`

Comment (tehsphinx) [reply languages/go/config.json:73]: Right... 👍

**Comment by tehsphinx** [comment]
@oanaOM: can you have a look again? Feel free to merge if it's ok.

**Comment by ErikSchierboom** [comment]
> I'd rather have a small paragraph on floats in the intro, instead.

As long as it is very minimal, this could work. Just enough for the student to solve the exercise.

**Pull request merged**